buildpack.toml

```diff
@@ -30,6 +30,10 @@ dir = "vendor/ruby/heroku-20"
 url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-22/ruby-3.1.4.tgz"
 dir = "vendor/ruby/heroku-22"
 
+[[publish.Vendor]]
+url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-22/amd64/ruby-3.1.4.tgz"
```

Comment:
```suggestion
url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-24/amd64/ruby-3.1.4.tgz"
```

Reply:
Good catch. I updated it locally and added a test that should catch this type of mistake going forward.